src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Skia.Gtk/MyExtensionsApp.Skia.Gtk.csproj

```diff
@@ -5,7 +5,7 @@
 		<TargetFramework>$baseTargetFramework$</TargetFramework>
 		<ApplicationIcon>Images\logo.ico</ApplicationIcon>
 	</PropertyGroup>
-	<!--#if winAppSdk-->
+	<!--#if useWinAppSdk-->
```

Comment:
I'm not sure why there's this linkage to the windows manifest file and I think it makes more sense to just have a separate manifest file for the gtk app